### ISSUE TITLE
Host pregame screen

### ISSUE THREAD

**Issue by I3uckwheat** [I3uckwheat/cards-of-carousal#109]
This screen is responsible for the pre-game, it should allow the user to select card packs, show the rules/how to play, show the joinCode, show a live list of players, and the settings menu in a modal. This component will also require setting some state in the HostContext to actually prep for the next part of the game.

This component should use the HostLayout to accomplish this.

![image](https://user-images.githubusercontent.com/4411701/110176919-b3995100-7dc9-11eb-830a-c5b06d3f8a65.png)
![image](https://user-images.githubusercontent.com/4411701/110176947-c14ed680-7dc9-11eb-970f-c44c2028962a.png)

# User Stories
* As a host, I would like to have a screen to allow people to join the game before we start the game so we can all start on equal footing
* As a host, I would like to be able to change settings before starting the game
* As a host, I would like pressing the "Start Carousing" button to download the card packs and store them on the Host context.
* As a host, I would like pressing the "Start Carousing" button to deal cards to the player and move to the next screen.
* As a host, I would like to see which players have joined the game, so I can tell when to start
* As a player, I would like to be able to see the join code on the host screen
* As a player, when the state changes to "select-white-cards" on the host, I would like to either see a message screen telling me I'm the Czar, or a screen with my hand to select the next cards

# Completion Criteria
- [x] Display the live user-list on the left panel
- [x] have the settings modal available, (it doesn't have to work yet that functionality is coming in another ticket)
- [x] Display the join-code
- [x] Allow for changing of settings by setting state in the HostContext actions can use as conditionals 
- [x] Clicking "Start Carousing" sets the `gameState` to `"waiting-for-deck"` (The screen should not change, may require modifications in the `HostScreenController`)
- [x] The Czar is chosen via a reducer action at random or incremented to the next player in the list.
- [x] The "Close Game" button takes the host back to the Welcome screen (could just call an action to disconnect players and refresh the screen)
- [x] The copy in the "how to play" section is displayed. (for now should be lorum ipsum)
